docs/docs/concepts/low_level.md

```diff
@@ -322,6 +322,63 @@ def continue_to_jokes(state: OverallState):
 graph.add_conditional_edges("node_a", continue_to_jokes)
 ```
 
+## `GraphCommand`
+
+It can be useful to combine control flow (edges) and state updates (nodes). For example, you might want to BOTH perform state updates AND  decide which node to go to next in the SAME node. LangGraph provides a way to do so by returning a [`GraphCommand`][langgraph.graph.state.GraphCommand] object from node functions:
+
+```python
+def my_node(state: State) -> GraphCommand[Literal["my_other_node"]]:
+    return GraphCommand(
+        # state update
+        update={"foo": "bar"},
+        # control flow
+        goto="my_other_node"
+    )
+```
+
+`GraphCommand` has the following properties:
+
+| Property | Description |
+| --- | --- |
+| `graph` | Graph to send the command to. Supported values:<br>- `None`: the current graph (default)<br>- `GraphCommand.PARENT`: parent graph |
+| `goto` | Name of the node to navigate to next. Can be any node that belongs to the specified `graph` (current or parent). If `goto` not specified, the graph will halt after executing the current superstep. |
```

Comment:
`If `goto` not specified, the graph will halt after executing the current superstep.` this is only true if there are no other edges out of this node, and no other nodes running at same time navigated elsewhere

Reply:
yup good point will clarify!